packages/retail-ui-extensions/src/components/Screen/Screen.ts

```diff
@@ -1,8 +1,20 @@
 import {createRemoteComponent} from '@remote-ui/core';
 
+export interface ScreenSearchBarProps {
+  initialValue?: string;
+  onTextChange?: (value: string) => void;
+  onSearch: (value: string) => void;
+  onFocus?: () => void;
+  visible?: boolean;
+  editable?: boolean;
+  placeholder?: string;
+  showBarcodeIcon?: boolean;
+}
+
 export interface ScreenProps {
   name: string;
   isLoading?: boolean;
+  searchBar?: ScreenSearchBarProps;
```

Comment:
I'm guessing undefined here will be the default, and not render a search bar in this case?